.ibm/pipelines/windows-test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash 
+
+###################################################
+# This script is used to run the test on windows
+# using the IBM DevOps Services.
+#
+
+set -x
+
+LOGFILE="pr-${GIT_PR_NUMBER}-windows-tests-${BUILD_NUMBER}"
+export REPO=${REPO:-"https://github.com/redhat-developer/odo"}
+#copy test script inside /tmp/
+sshpass -p $WINDOWS_PASSWORD scp -o StrictHostKeyChecking=no ./.ibm/pipelines/windows-test-script.ps1    Administrator@161.156.170.160:/tmp/windows-test-script.ps1
+
+#execute test from the test script
+sshpass -p $WINDOWS_PASSWORD ssh Administrator@161.156.170.160 -o StrictHostKeyChecking=no powershell /tmp/windows-test-script.ps1 "${GIT_PR_NUMBER}" "${BUILD_NUMBER}" "${API_KEY}" "${IBM_OPENSHIFT_ENDPOINT}" "${LOGFILE}" "${REPO}"
```

Comment:
Could you use a variable for the IP of the windows machine, and pass this variable from the IBM Pipeline config (similarly to clusters URLs / IDs)?

Reply:
Understood